aviary/validation_cases/benchmark_tests/test_bench_FwFm.py

```diff
@@ -286,7 +286,7 @@ def bench_test_swap_4_FwFm(self):
                     "initial_bounds": ((0.0, 2.0), "min"),
                     "duration_bounds": ((5.0, 50.0), "min"),
                     "no_descent": False,
-                    "add_initial_mass_constraint": False,
+                    "add_initial_mass_constraint": True,
```

Comment:
why did this change?

Reply:
better opt convergence posing it as a constraint instead of a connection